index.html

```diff
@@ -197,6 +196,27 @@ <h2>About Myself</h2>
         </ul>
       </div>
     </section>
+    <!-- Form -->
+    <section id="form" class="flex">
+      <form class="flex-column" action="#" method="post">
```

Comment:
- [ ] Your form should post data using [form spree](https://formspree.io/html/). Please follow the link provided and make sure your form posts to from spree. This is also a project requirement